app/controllers/questions/work_situations_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Questions
+  class WorkSituationsController < QuestionsController
+    def self.show?(intake)
+      intake.had_a_job?
+    end
```

Comment:
Does this have a unit test?

Reply:
None of the other controllers used this criteria to determine whether to show themselves, as far as I can tell. Did we trule want to gate all these questions on whether they indicated having 1 or more jobs?

Reply:
This would mean if someone said they didn't have a job, then we wouldn't ask them if they have unemployment income. It doesn't seem correct.

Reply:
i'll discuss your last point with Anu. this was the intent of the design, but sounds like it might not be right!

Reply:
thought i worked in a test too but will double check. might need to change this logic anyway now per your last point.

Reply:
the unit test was in work_situations_controller_spec, but you were correct about the logic issue with the unemployment question so i'm submitting a change that will remove the skip logic (and the test)